shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/PostgreSQLKeyword.g4

```diff
@@ -587,10 +587,6 @@ TSQUERY
     : T S Q U E R Y
     ;
 
-UUID
```

Comment:
UUID is a key word for data tyep in PostgreSQL. https://www.postgresql.org/docs/9.1/datatype-uuid.html

Reply:
There is no uuid keyword in the source code.
https://github.com/postgres/postgres/blob/REL_12_STABLE/src/include/parser/kwlist.h